### PR TITLE
[XM] Fix dontlink by fixing some NSAccessibility binding mistakes

### PR DESCRIPTION
**Pull request by chamons**
This commit was not run against dontlink and sadness occurred: https://github.com/xamarin/xamarin-macios/commit/f535df6168690e7016460fc6f0b7b73bcbca0c71

This fixes things up.

I did not want to expose the previously named NSObjectAccessibilityExtensions on NSObject, even if that's what the header says, since that will really dirty up auto complete. Oh and:

   BOOL oDoes = [o respondsToSelector:@selector(accessibilityHitTest:)]; // NSObject * o - NO
   BOOL vDoes = [v respondsToSelector:@selector(accessibilityHitTest:)]; // NSView * v - YES

So I did a copy/paste. Which is terrible. Not sure a better way (open to suggestions).

### CODE REVIEW

File: src/appkit.cs [hunk 1]
```diff
@@ -22049,7 +22049,7 @@ partial interface NSAccessibilityAttributes {
 		[Field ("NSAccessibilityDisclosedRowsAttribute")]
 		NSString DisclosedRowsAttribute { get; }
 
-		[Field ("NSAccessibilityDisclosedByRowsAttribute")]
+		[Field ("NSAccessibilityDisclosedByRowAttribute")]
 		NSString DisclosedByRowsAttribute { get; }
```
Comment (spouliot) [inline src/appkit.cs]: Managed name should be fixed too (extra `s`)

File: src/appkit.cs [hunk 2]
```diff
@@ -22368,7 +22368,7 @@ public interface NSAccessibilitySubroles {
 		NSString DefinitionListSubrole { get; }
 
 		[Mac (10, 7)]
-		[Field ("NSAccessibilityFullscreenButtonSubrole")]
+		[Field ("NSAccessibilityFullScreenButtonSubrole")]
 		NSString FullscreenButtonSubrole { get; }
```
Comment (spouliot) [inline src/appkit.cs]: same

**Comment by spouliot** [comment]
IIRC you can just create an interface **without any attribute** and have it inlined (or I dreamt that feature).

**Comment by chamons** [comment]
@spouliot I could have sworn I tried that, but testing suggests I didn't. Let me rework...

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by spouliot** [comment]
LGTM if dontlink-mac is happy :)

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by rolfbjarne** [comment]
LGTM.

**Pull request merged**